security/src/main/java/org/frankframework/lifecycle/servlets/ServletAuthenticatorBase.java

```diff
@@ -209,6 +215,27 @@ public SecurityFilterChain configureHttpSecurity(HttpSecurity http) {
 		}
 	}
 
+	// Match when the client matches '<servlet-path>/logout'.
+	private boolean requestMatcher(HttpServletRequest request) {
+		return ("GET".equals(request.getMethod()) && "/logout".equals(request.getPathInfo()));
```

Comment:
What would be the servlet path here?
`/iaf/logout` or `/logout`?

Reply:
Servlet mapping, so `iaf/api/` + `logout` or `iaf/gui/` + `logout`.
In a new PR I want to move this, as well as CSRF (and session management) to a different configurer for the Console, so both endpoints (frontend/backend) are seen as 1, as well as not using all this silly stuff for all web-service endpoints.

Reply:
Okay, so angular also tries to work with a route on `/iaf/gui/#/logout`
This shouldnt be an issue but it could be, maybe I should change that route in a next pr so I can be sure it wont ever conflict